shared/common-adapters/user-card.native.tsx

```diff
@@ -70,7 +70,7 @@ const styles = Styles.styleSheetCreate(() => ({
   inside: {
     ...Styles.globalStyles.flexBoxColumn,
     alignItems: 'stretch',
-    backgroundColor: Styles.globalColors.fastBlank,
+    backgroundColor: Styles.globalColors.blueGrey,
```

Comment:
Is this needed? Other places this is used has a blue background from the container; I think this'd be unexpected to have on this component.

Reply:
So you'd just ditch `backgroundColor`?

Reply:
Yeah